ZuluContent/Items/Skill Items/Magical/Misc/Moongate.cs

```diff
@@ -4,54 +4,35 @@
 using Server.Network;
 using Server.Gumps;
 using Server.Regions;
+using Server.Spells;
```

Comment:
Never realised you could Dispel moongates before lmao!

Reply:
Haha yeah you would use it to stop people following you